src/ert/ensemble_evaluator/_ensemble.py

```diff
@@ -303,7 +303,10 @@ async def _evaluate_inner(  # pylint: disable=too-many-branches
 
             self._scheduler.add_dispatch_information_to_jobs_file()
             result = await self._scheduler.execute(min_required_realizations)
-
+        except PermissionError as error:
+            logger.exception((f"Unexpected exception in ensemble: \n {error!s}"))
+            await event_unary_send(event_creator(Id.ENSEMBLE_FAILED))
```

Comment:
Now when this gets propagated I'd expect that you will check such status in the test.(?)

Reply:
This was leftovers from 90c14a927fb0e4c1de9e08a88f7599f329342a0a that was included due to some git wizardry 🧙‍♂️.
I will add some tests for it in a different PR  🌴 🌞